From 3e68166fe8c7cad8c3fc777d8b4f77215789f5c8 Mon Sep 17 00:00:00 2001
From: Matt Selsky <selsky@columbia.edu>
Date: Thu, 11 Nov 2010 15:32:51 -0800
Subject: [PATCH] Bug: 3336

Update list of supported releases and fix URLs for download sites.
---
 README |   16 ++++++----------
 1 files changed, 6 insertions(+), 10 deletions(-)

diff --git a/README b/README
index 7053b81..4240770 100644
--- a/README
+++ b/README
@@ -1,14 +1,10 @@
-$Id: README,v 1.15 2010/01/06 17:01:26 murch Exp $
-
 This is the Carnegie Mellon University Cyrus IMAP Server.
 
-This distribution is what we are running on our production servers.  
-
-The 2.2, 2.1 and 2.0 releases contains many new features over 1.6.
+The 2.4 release contains many new features over previous major releases.
 
-No further development work will progress on 1.6 so we encourage you
-to work with the Cyrus 2.2 code wherever possible.  Version 1.6
-has known security vulnerabilities, and is not supported.
+No further development work will be done on 2.2.x or old releases.  We
+encourage you to upgrade to the latest Cyrus 2.3.x or 2.4.x releases,
+wherever possible.
 
 If you continue to install this software PLEASE be sure to read the
 documentation (located in the doc/ subdirectory).  Many common questions
@@ -26,7 +22,7 @@ the binary distribution.
   read doc/readme.html first!).
 
 * To obtain updates to this software, visit the web page
-            http://cyrusimap.web.cmu.edu/downloads.html
+            http://www.cyrusimap.org/releases/
   or ftp directly from
-            ftp://ftp.andrew.cmu.edu/pub/cyrus-mail
+            ftp://ftp.cyrusimap.org/cyrus-imapd/
 
-- 
1.7.3.2

